src/components/common/DropdownItem.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'proptypes';
+import classNames from 'classnames';
+import Checkbox from './Checkbox';
+
+const DropdownItem = ({
+  label,
+  value,
+  active,
+  handleClick,
+}) => {
+  const itemClassName = classNames('dropdown-item', {
+    'is-active': active,
+  });
+
+  const itemWidth = '350px';
```

Comment:
can this be a prop too?

Reply:
Added `width` as required prop to DropdownItem. Items will receive width from Dropdown (either default Dropdown value or whatever value is passed in props).